backend/src/main/java/org/eclipse/tractusx/semantics/hub/bamm/BammHelper.java

```diff
@@ -154,4 +157,31 @@ public Try<String> getExamplePayloadJson(Aspect aspect) {
 
         return Try.of(payloadGenerator::generateJson);
     }
+
+    public Try getAasSubmodelTemplate(Aspect aspect, String aasFormat) {
+        AspectModelAASGenerator aasGenerator = new AspectModelAASGenerator();
+        ByteArrayOutputStream stream = new ByteArrayOutputStream();
+        
+        try {
+            switch(aasFormat) {
+                case "file":
+                    aasGenerator.generateAASXFile(aspect, (String s) -> {
+                        return stream;
+                    });
+                    stream.close();
```

Comment:
Closing the ByteArrayOutputStream does not have any effect. You can remove it and I think the try catch as well.

Reply:
With the close I think you are right. However, the try catch block is necessary due to a potential IOException.